src/wheel-zoom.js

```diff
@@ -190,6 +190,10 @@ WZoom.prototype = {
             content.originalHeight = options.height || content.$element.offsetHeight;
         }
 
+        const scale = content.$element.style.scale;
+        content.originalScale = scale ? scale.split(" ").map(p => parseFloat(p)) : [1,1,1];
+        content.originalTranslateZ = content.$element.style.translate?.split(" ")[2] || "0px";
```

Comment:
Pls, help me understand the reason for introducing these properties( 

CSS are my weak point(